frontend/simple/views/utils/validators.js

```diff
@@ -1,5 +1,13 @@
+import { HashableGroupProposal } from '../../../../shared/events.js'
+
 export const nonWhitespace = (value: string) => /^\S+$/.test(value)
 
 export const decimals = (digits: number) => (value: number) => Number.isInteger(value * Math.pow(10, digits))
 
-export const votingType = (type: String) => ['mincome', 'rule', 'member'].includes(type)
+export const votingType = (type: String) => [HashableGroupProposal.TypeInvitation, HashableGroupProposal.TypeRemoval, HashableGroupProposal.TypeIncome, HashableGroupProposal.TypeChangeThreshold, HashableGroupProposal.TypeApprovalThreshold, HashableGroupProposal.TypeRemovalThreshold].includes(type)
+
+export const votesObj = (votes: String) => ( // total, received, threshold
+  votes.total && Number.isInteger(votes.total) &&
+  votes.received && Number.isInteger(votes.received) &&
+  votes.threshold && !isNaN(votes.received)
```

Comment:
Can you explain me why `!isNaN(votes.received)` ?

Reply:
!isnan is just checking if this is a number, and I ended up re-getting the threshold from the store instead of passing it, so great catch, will remove! :)